requirements.txt

```diff
@@ -1,19 +1,64 @@
+aiosmtplib==2.0.2
 annotated-types==0.5.0
 anyio==3.7.1
+bank-statement-reader-altara==1.3.6
+blinker==1.6.2
+boto3==1.28.22
+botocore==1.31.22
+cffi==1.15.1
+charset-normalizer==3.2.0
 click==8.1.6
+cryptography==41.0.3
+dnspython==2.4.2
+email-validator==2.0.0.post2
+et-xmlfile==1.1.0
 fastapi==0.100.1
+fastapi-mail==1.4.1
+greenlet==2.0.2
 gunicorn==21.2.0
 h11==0.14.0
 httptools==0.6.0
 idna==3.4
+iniconfig==2.0.0
+iso3166==2.1.1
+Jinja2==3.1.2
```

Comment:
Why do we need jinja?

Reply:
Not me; it came with the framework. I am guessing its a dependency on  fastapi-mail

I checked sha, it is for templating, so it will be useful for the email-sending template